### PR TITLE
Error, ManageState, Notification snaps - integration tests

### PR DESCRIPTION
**Pull request by irubido**
Error,
ManageState,
Notification
integration tests

### CODE REVIEW

**Review by mpetrunic — APPROVED** [review]

File: packages/error/package.json [hunk 1]
```diff
@@ -39,6 +39,7 @@
     "@metamask/snap-types": "^0.23.0"
   },
   "devDependencies": {
+    "@chainsafe/dappeteer": "^4.0.0-rc.0",
```
Comment (BeroBurny) [inline packages/error/package.json]:
```suggestion
    "@chainsafe/dappeteer": "^4.0.0-rc.1",
```

File: packages/manageState/package.json [hunk 2]
```diff
@@ -36,6 +36,7 @@
     "@metamask/snap-types": "^0.23.0"
   },
   "devDependencies": {
+    "@chainsafe/dappeteer": "^4.0.0-rc.0",
```
Comment (BeroBurny) [inline packages/manageState/package.json]:
```suggestion
    "@chainsafe/dappeteer": "^4.0.0-rc.1",
```

File: packages/notification/package.json [hunk 3]
```diff
@@ -36,6 +36,7 @@
     "@metamask/snap-types": "^0.23.0"
   },
   "devDependencies": {
+    "@chainsafe/dappeteer": "^4.0.0-rc.0",
```
Comment (BeroBurny) [inline packages/notification/package.json]:
```suggestion
    "@chainsafe/dappeteer": "^4.0.0-rc.1",
```

**Review by BeroBurny — COMMENTED** [review]
Looking good but please keep libraries to the latest version

**Comment by irubido** [comment]
all tests
https://github.com/ChainSafe/test-snaps/pull/3

**Pull request closed without merging**